playbook-website/app/javascript/site_styles/_site-style.scss

```diff
@@ -4,6 +4,9 @@
 @import "playbook-ui/dist/tokens/border_radius";
 @import "playbook-ui/dist/tokens/shadows";
 
+//must bring leaflet css in via cdn to avoid webpack parsing issues
+@import url("https://unpkg.com/leaflet@1.9.3/dist/leaflet.css");
```

Comment:
Leaflet CSS does not play well with webpack 4. Bringing the CSS in through the dist throws several parsing errors. Bringing it in via CDN fixes all those issues and is also the methods of consumption recommended by Leaflet itself.